perun-core/src/main/java/cz/metacentrum/perun/core/api/AttributesManager.java

```diff
@@ -1733,7 +1733,7 @@ Map<String, Attribute> getEntitylessAttributesWithKeys(PerunSession sess, String
 	 * @throws InternalErrorException if an exception raise in concrete implementation, the exception is wrapped in InternalErrorException
 	 * @throws PrivilegeException     if privileges are not given
 	 */
-	void deleteAttribute(PerunSession sess, AttributeDefinition attributeDefinition) throws PrivilegeException, AttributeNotExistsException;
+	void deleteAttribute(PerunSession sess, AttributeDefinition attributeDefinition) throws PrivilegeException, AttributeNotExistsException, RelationExistsException;
```

Comment:
Maybe should be `RelationExistsException` also mentioned in `AttributesManagerMethod.java` in the doc for this method?